ui/src/pages/RepoHome.js

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, {Component} from 'react';
+import {FormattedHTMLMessage, FormattedMessage, defineMessages, injectIntl} from 'react-intl';
+import Button from '@material/react-button';
+
+import Footer from './../components/Footer.js';
+import LoadingIndicator from './../components/LoadingIndicator.js';
+import RepoHeader from './../components/RepoHeader.js';
+import SearchBar from './../components/SearchBar.js';
+
+const messages = defineMessages({
+  provideInfoAboutSomeone: {
+    id: 'RepoHome.provideInfoAboutSomeone',
+    defaultMessage: 'Provide information about someone',
+    description: ('Label on a button for people who want to provide '
+        + 'information about someone\'s status in the aftermath of a '
+        + 'disaster.'),
+  },
+  or: {
+    id: 'RepoHome.or',
+    defaultMessage: 'Or',
+    description: ('A heading for a section with one or more alternatives to '
+        + 'the main option.'),
+  },
+  repoRecordCount: {
```

Comment:
I thought it might be more convenient to have messages defined in the same file where they're used; I definitely don't have a strong preference about that though, so if you'd prefer to have messages defined centrally I'm happy to move them. Alternatively, we could have a CommonMessages.js file or something like that for the messages used in multiple places, and keep the others in the file where they're used.
However, I think that, no matter where we define them, we're going to have to have a way to de-dupe messages since there's going to be a lot of overlap between this and the Lite/AMP mode.

Reply:
> I thought it might be more convenient to have messages defined in the same file where they're used; I definitely don't have a strong preference about that though, so if you'd prefer to have messages defined centrally I'm happy to move them. Alternatively, we could have a CommonMessages.js file or something like that for the messages used in multiple places, and keep the others in the file where they're used.

Either of the approaches here sounds good to me. I have no strong preference among them as far as a common message is defined just once.

> However, I think that, no matter where we define them, we're going to have to have a way to de-dupe messages since there's going to be a lot of overlap between this and the Lite/AMP mode.

That's a good point. Maybe we should figure out how to define a message across JS and Python.

Reply:
I realized the two messages are actually slightly different: this one starts with the word "currently", whereas the message on the global homepage doesn't (maybe because of space concerns?).

Reply:
I see, then that's OK. May be worth asking Travis if it's intentional though.

Reply:
Travis seems to be at a conference and I haven't heard back yet, so I added a todo here.